sphericart-jax/python/tests/test_autograd.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import jax
+jax.config.update("jax_platform_name", "cpu")
+jax.config.update("jax_enable_x64", True)
```

Comment:
This might fail on M1/ARM

Reply:
Ok! I'll take them out except for the test that checks precision, where it is necessary